### ISSUE TITLE
./.libs/libkvazaar.a(libavx2_la-ipol-avx2.o):ipol-avx2.c:(.text+0x2163): undefined reference to `_mm_extract_epi64'

### ISSUE THREAD

**Issue by Selur** [ultravideo/kvazaar#214]
Using MABS (https://github.com/jb-alvarado/media-autobuild_suite) to build Kvazaar on Windows I get

```
./.libs/libkvazaar.a(libavx2_la-ipol-avx2.o):ipol-avx2.c:(.text+0x2163): undefined reference to `_mm_extract_epi64'
./.libs/libkvazaar.a(libavx2_la-ipol-avx2.o):ipol-avx2.c:(.text+0x218a): undefined reference to `_mm_extract_epi64'
./.libs/libkvazaar.a(libavx2_la-ipol-avx2.o):ipol-avx2.c:(.text+0x21b0): undefined reference to `_mm_extract_epi64'
./.libs/libkvazaar.a(libavx2_la-ipol-avx2.o):ipol-avx2.c:(.text+0x248a): more undefined references to `_mm_extract_epi64' follow
collect2.exe: error: ld returned 1 exit status
make[2]: *** [Makefile:965: kvazaar.exe] Error 1
make[2]: Leaving directory '/build/kvazaar-git/build-32bit/src'
make[1]: *** [Makefile:524: all-recursive] Error 1
make[1]: Leaving directory '/build/kvazaar-git/build-32bit'
make: *** [Makefile:394: all] Error 2
```

**Comment by Arizer** [comment]
Hi,

Thanks for reporting. Our automated tests also reported this (when the queue at last cleared...). I'll probably fix this next monday. Only the 32-bit windows build is broken as far as I know.

**Comment by Arizer** [comment]
Fixed (a832206)
